test/integration/server-selection/server_selection.spec.test.ts

```diff
@@ -3,18 +3,19 @@ import * as path from 'path';
 import { loadSpecTests } from '../../spec';
 import { runUnifiedSuite } from '../../tools/unified-spec-runner/runner';
 
-describe.skip('Server Selection Unified Tests (Spec)', function () {
+describe('Server Selection Unified Tests (Spec)', function () {
   const tests = loadSpecTests(path.join('server-selection', 'logging'));
   runUnifiedSuite(tests, test => {
     if (
       [
         'Failed bulkWrite operation: log messages have operationIds',
-        'Successful bulkWrite operation: log messages have operationIds'
+        'Failed client bulkWrite operation: log messages have operationIds',
+        'Successful bulkWrite operation: log messages have operationIds',
+        'Successful client bulkWrite operation: log messages have operationIds'
```

Comment:
Old bulkWrite we knew we didn't have operationId support but by nature of copying out Node.js specific versions of these tests did we miss adding support to client.bulkWrite too? I have not looked into what it is or does, so maybe it doesn't fit into our API well.